src/components/illustrationList/illustrationList.module.css

```diff
@@ -0,0 +1,23 @@
+:not(.illustrationlist) + .illustrationList {
```

Comment:
Ah. Nå ser jeg det. Feil casing
```suggestion
:not(.illustrationList) + .illustrationList {
```